src/api/cvc4cpp.h

```diff
@@ -523,6 +523,122 @@ struct CVC4_PUBLIC SortHashFunction
   size_t operator()(const Sort& s) const;
 };
 
+/* -------------------------------------------------------------------------- */
+/* Op                                                                     */
+/* -------------------------------------------------------------------------- */
+
+/**
+ * A CVC4 operator term.
+ * An operator term is a term that represents certain operators, instantiated
+ * with its required parameters, e.g., a term of kind BITVECTOR_EXTRACT.
```

Comment:
This still says `operator term` -- should we change this to `operator`?

Reply:
Oops, yep we should -- missed it with `sed`. Fixed with d19d4b3a2465d44ca8372af2676a0e902a47f654